test/install_script/nvm_detect_profile

```diff
@@ -27,19 +27,35 @@ SHELL="/bin/bash"
 
 # $SHELL is set to bash and .bashrc is there, it must be detected
 _PROFILE=$(nvm_detect_profile)
-[ "_$_PROFILE" = "_$HOME/.bashrc" ] || echo "_\$HOME/.bashrc: _$HOME/.bashrc\n" \
-                                       echo "_\$_PROFILE: _$_PROFILE\n" \
-                                       die "nvm_detect_profile didn't pick $SHELL and $HOME/.bashrc"
+[ "_$_PROFILE" = "_$HOME/.bashrc" ] || ( echo "_\$HOME/.bashrc: _$HOME/.bashrc" &&
+                                         echo "_\$_PROFILE: _$_PROFILE" &&
+                                         die "nvm_detect_profile didn't pick $SHELL and $HOME/.bashrc" )
+
+# But $PROFILE should override
+_PROFILE="$(PROFILE=test_profile nvm_detect_profile)"
+_PROFILE=$(nvm_detect_profile)
+[ "_$_PROFILE" = "_$PROFILE" ] || ( echo "_\$_PROFILE: _$_PROFILE" &&
```

Comment:
this reads pretty confusing to me - could we rename `_PROFILE` to `DETECTED_PROFILE` or something?